src/common.js

```diff
@@ -0,0 +1,29 @@
+import { maxSatisfying } from 'es-semver';
+
```

Comment:
You could also add here the paths `load.js` and `style.css` which are used in Hapi `start-server.js` and Cloudflare `index.js`

Reply:
Wow, awesome idea :heart_eyes:

Reply:
Done

Reply:
I added them to their respective files (not the common one).